aten/src/ATen/cpu/vec256/functional.h

```diff
@@ -231,4 +231,33 @@ inline void map3(
   }
 }
 
+template <typename scalar_t, typename Op>
+inline void map4(
```

Comment:
Please add vec256 tests

Reply:
Test case added in `aten/src/ATen/test/vec256_test_all_types.cpp`